README.md

```diff
@@ -1 +1,219 @@
+# Protocol
+
+With Bloock Identity, developers can empower users to exchange credentials that are securely verified using advanced cryptography and blockchain technology. Tailored for developers, Bloock Identity prioritizes privacy, decentralization, and user data self-sovereignty.
+
+Bloock Identity follows the [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/) standard and [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/) standard.
+
+## The basics
+
+### What do we mean by Identity?
+An identity can be a person, a company, an organization, a DAO, or a government. Identity can even be a thing: a chair, a room, a bot, and so on. When we talk about identities, we are referring to "identities as accounts".
+
+### What is a claim?
+An Identity can provide a claim. You can think of a claim as a statement: something an Identity is saying.
+
+Most of the time, these statements refer to other identities. In other words, **claims usually create relations between identities.**
+
+For example, when a university (identity) says that a student (identity) has a degree, this is a statement (claim) about the student. This statement creates a relation between the student and the university.
+
+### What are the agents in this process?
+- **Issuer**: An actor who makes a claim.
+- **Holder**: An actor who receives a claim.
+- **Verifier**: An actor who verifies if the content of a claim is issued by a specific identity and held by another specific identity.
+
+### And therefore, what is a verifiable credential?
+**Credential**: Data that is needed to prove that a claim is issued by a specific identity and held by another specific identity. This data is composed of **a claim and a proof**.
```

Comment:
A Verifiable Credential is the set of data that represents the statement made from an issuer to a holder and information needed to make this statement verifiable.
It usually contains:
- Information to identify the issuer
- Information to identity the holder
- The credential type or schema
- The attributes or properties being asserted by the issuer
- Evidences that can be used to verify the integrity and/or authenticity of the credential
- Other information such as validity period, terms, ...